src/table/table.component.tsx

```diff
@@ -168,6 +262,12 @@ const Table = React.memo((props: TableProps): React.ReactElement => {
             page={page}
             rowsPerPage={resultsPerPage}
           />
+          <ColumnCheckboxes
+            availableColumns={availableColumns}
+            checkedColumns={checkedColumns}
+            onColumnOpen={handleColumnOpen}
+            onColumnClose={handleColumnClose}
+          />
```

Comment:
Should this really be in the table component? In the future, this will be a separate epic and thus I don't think it should be in `Table`. However, by then I'd expect we'd have redux and perhaps that will result in this not having to be nested in `Table` as well...

Reply:
My reasoning is that we can't have one without the other so they're intrinsically linked. If we have checkboxes, even if none of them are selected, we still have a table element and the table will always have an ID column loaded, for example